### DISCUSSION TITLE
configure failed

### DISCUSSION THREAD

**Discussion by andrewzr** [apache/apisix#5073] [Q&A] [answered]
Hi, was checking out the documentation of installing dependencies of apisix 2.9 with this: https://apisix.apache.org/docs/apisix/install-dependencies

everything was good till installing the Luarocks script:
curl https://raw.githubusercontent.com/apache/apisix/master/utils/linux-install-luarocks.sh -sL | bash -
<img width="767" alt="Screenshot 2021-09-16 at 14 18 10" src="https://user-images.githubusercontent.com/72366277/133559960-346c1d95-611b-4e32-8c63-0a7bde802489.png">

tested the script on debian 10 and then on ubuntu 16.04 LTS using a VM and both shows configure failed

is there anything i missed out?

**Comment by dmsolr** [comment, marked as answer]
The picture is illustrated very clearly. 
**could not find unzip**
Please install unzip and retry.
